app/src/main/kotlin/io/eugenethedev/taigamobile/ui/screens/profile/ProfileScreen.kt

```diff
@@ -0,0 +1,337 @@
+                contentAlignment = Alignment.Center
+            ) {
+                CircularLoader()
+            }
+        }
+        else -> {
+            ProfileScreenContent(
+                currentUser = currentUser,
+                currentUserStats = currentUserStats,
+                currentUserProjects = currentUserProjects,
+            )
+        }
+    }
+}
+
+@OptIn(ExperimentalCoilApi::class)
+@Composable
+fun ProfileScreenContent(
+    currentUser: User? = null,
+    currentUserStats: Stats? = null,
+    currentUserProjects: List<Project> = emptyList()
+) {
+    Image(
+        painter = rememberImagePainter(
+            data = currentUser?.avatarUrl ?: R.drawable.default_avatar,
+            builder = {
+                error(R.drawable.default_avatar)
+                crossfade(true)
+            },
+        ),
+        contentDescription = null,
+        contentScale = ContentScale.Crop,
+        modifier = Modifier
+            .size(200.dp)
+            .clip(CircleShape)
+    )
+
+    Spacer(Modifier.height(16.dp))
+
+    Text(
+        text = currentUser?.fullName ?: "Full name",
+        style = MaterialTheme.typography.titleLarge
+    )
+
+    Spacer(Modifier.height(2.dp))
+
+    Text(
+        text = "@${currentUser?.username ?: "username"}",
+        color = MaterialTheme.colorScheme.outline,
+        style = MaterialTheme.typography.bodyLarge,
+    )
+
+    Spacer(Modifier.height(24.dp))
+
+    Row(
+        horizontalArrangement = Arrangement.SpaceEvenly,
+        modifier = Modifier.fillMaxWidth()
+    ) {
+        ColumnTextData(currentUserStats?.totalNumProjects.toString(), stringResource(R.string.projects))
+        ColumnTextData(
+            currentUserStats?.totalNumClosedUserStories.toString(),
+            stringResource(R.string.closed_user_story)
+        )
+        ColumnTextData(currentUserStats?.totalNumContacts.toString(), stringResource(R.string.contacts))
+    }
+
+    Spacer(Modifier.height(24.dp))
+
+    Text(
+        text = stringResource(R.string.projects),
+        style = MaterialTheme.typography.titleLarge
+    )
+
+    Spacer(Modifier.height(12.dp))
+
+    LazyColumn(Modifier.padding(horizontal = mainHorizontalScreenPadding)) {
+        items(currentUserProjects) {
+            ProjectItem(it)
+            Spacer(Modifier.height(6.dp))
+        }
+
+        item {
+            Spacer(Modifier.navigationBarsHeight(8.dp))
+        }
+    }
+}
+
+@Composable
+private fun ColumnTextData(titleText: String, bodyText: String) = Column(
+    horizontalAlignment = Alignment.CenterHorizontally
+) {
+    Text(
+        text = titleText,
+        style = MaterialTheme.typography.titleMedium
+    )
+
+    Spacer(Modifier.height(2.dp))
+
+    Text(
+        text = bodyText,
+        color = MaterialTheme.colorScheme.outline,
+        style = MaterialTheme.typography.bodyLarge,
+    )
+}
+
+@Composable
+private fun ProjectItem(project: Project) = Row(
```

Comment:
Why don't reuse [this `@Composable function`](https://github.com/EugeneTheDev/TaigaMobile/blob/master/app/src/main/kotlin/io/eugenethedev/taigamobile/ui/screens/dashboard/DashboardScreen.kt#L162) (only without projects switching functionality)?. Make it component and reuse here